Makefile

```diff
@@ -81,10 +86,19 @@ localeextract:
 	python manage.py extract
 	python manage.py merge
 
+locale-populate-react:
+	@ mkdir -p locale/compendia
+	for localename in `find locale -mindepth 1 -maxdepth 1 -type d | cut -d/ -f2 | grep -v templates | grep -v compendia`; do \
```

Comment:
For me, this `find` command returns the `locale/.git` directory as well, which in turn causes:
```
msgcat: error while opening "locale/.git/LC_MESSAGES/django.po" for reading: No such file or directory
msgmerge: error while opening "locale/compendia/.git.compendium" for reading: No such file or directory
```
when I run `make locale-populate-react`.

I modified the `find` command like this:
```
find locale -mindepth 1 -maxdepth 1 -type d -not -path locale/.git
```
and that got rid of the issue, but you may know a more elegant way to exclude the `locale/.git` directory.